DockerEnv

```diff
@@ -0,0 +1,78 @@
+OPENVPN_PROVIDER=**None**
+OPENVPN_CONFIG=**None**
+OPENVPN_USERNAME=**None**
+OPENVPN_PASSWORD=**None**
+LOCAL_NETWORK=**None**
```

Comment:
Leaving this as default will probably fail on startup, right? It's not an ip address so I guess that would be tricky for the ip tables commands reading that variable.